templates/taxi/car_detail.html

```diff
@@ -1,8 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h3> 
+    Car:
+    <a style="float: right" href="{% url 'taxi:car-delete' pk=car.id %}" >Delete</a>
+    <br>
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on -- checklist